tests/lax_numpy_test.py

```diff
@@ -2325,6 +2325,38 @@ def testHistogram(self, shape, dtype, bins, density, weights):
                               tol=tol)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+    {"testcase_name": "_{}_bins={}_weights={}_density={}".format(
+      jtu.format_shape_dtype_string(shape, dtype), bins, weights, density),
+      "shape": shape,
+      "dtype": dtype,
+      "bins": bins,
+      "weights": weights,
+      "density": density
+    }
+    for shape in [(5,), (12,)]
+    for dtype in int_dtypes
+    # We only test explicit integer-valued bin edges because in other cases
+    # rounding errors lead to flaky tests.
```

Comment:
This comment doesn't match the code below (`bins=[2, 2]` is specifying a number of bins for each axis, rather than explicit integer-valued bin edges)

Reply:
Ah that's right.  I think I also made that mistake [here](https://github.com/google/jax/blob/cf1fefcdc65872786e7ed8aafdade35dd4fe8e6c/tests/lax_numpy_test.py#L2339), should I add that change to this PR?

Reply:
Sure, that sounds good.